File: 2 SEMESTRE/1 BIMESTRE/ORIENTAÇÃO A OBJETO/ATIVIDADE CLASSE/Atividade Classes/src/Endereco.cpp
#include "Endereco.h"
#include <string>

using namespace std;

Endereco::Endereco()
{
    numero = 1;
}

Endereco::Endereco(string c, string r, int num)
{
    cidade = c;
    rua = r;
    numero = num;

}

Endereco::~Endereco()
{
    //dtor
}

string Endereco::Getcidade(){
    return cidade;
}
void Endereco::Setcidade(string val){
    cidade = val;
}

string Endereco::Getrua(){
    return rua;
}
void Endereco::Setrua(string val){
    rua = val;
}

int Endereco::Getnumero(){
    return numero;
}
void Endereco::Setnumero(int val){
    numero = val;
}

File: 2 SEMESTRE/2 BIMESTRE/ORIENTAÇÃO A OBJETO/AulaPOO/src/aulapoo/Endereco.java

package aulapoo;

public class Endereco {
    
    private String cidade;
    private String rua;
    private int numero;
    
   public Endereco(){
       
   }
   
   public Endereco(String cidade, String rua, int numero){
       this.cidade = cidade;
       this.rua = rua;
       this.numero = numero;
   }

    public String getCidade() {
        return cidade;
    }

    public String getRua() {
        return rua;
    }

    public int getNumero() {
        return numero;
    }

    public void setCidade(String cidade) {
        this.cidade = cidade;
    }

    public void setRua(String rua) {
        this.rua = rua;
    }

    public void setNumero(int numero) {
        this.numero = numero;
    }
   
   
    
}
